go-controller/pkg/ovnwebhook/nodeadmission.go

```diff
@@ -25,7 +25,7 @@ type checkNodeAnnot func(v annotationChange, nodeName string) error
 
 // commonNodeAnnotationChecks holds annotations allowed for ovnkube-node:<nodeName> users in non-IC and IC environments
 var commonNodeAnnotationChecks = map[string]checkNodeAnnot{
-	util.OvnNodeHostAddresses:            nil,
+	util.OVNNodeHostCIDRs:                nil,
```

Comment:
I don't know the order of upgrades for the new node-identity component. My question is: Is there a chance that this new component shows up alongside 4.13 ovnkube-node's? Or will the new component show up only after all ovnkube-node's are in 4.14. Basically do we need to handle the presence of an old ovnkube-node that could potentially update the old annotation and fail due to webhooks not knowing about that annotation?